vault/blog.2021.2021-09-27-markdown-on-mobile.md

```diff
@@ -1,66 +1,78 @@
+  - [Markdown notes](#markdown-notes)
+  - [Git clients](#git-clients)
+- [Last thoughts](#final-thoughts)
+
+## Obsidian
+
+![Editor and markdown-preview screenshots, on Android, for Obsidian](https://org-dendron-public-assets.s3.amazonaws.com/images/blog-obsidian-screenshots.png)
+
+> _"Obsidian is a powerful knowledge base on top of a local folder of plain text Markdown files."_
+
+- **Cost:** Free (or [paid options](https://obsidian.md/pricing))
+- **OS:** Android, iOS / iPad / Mac, Windows
+- **Open Source:** No
+- **Built-in git integration:** No
+
+_[Obsidian](https://obsidian.md/mobile)_ is a PKM (Personal Knowledge Management) tool that provides a sleek UI to Markdown files. It had the best mobile experience for me, on Android, out of the apps I tested. Though it doesn't have git support built-in, other mobile apps can assist in providing that backend. I recommend [MGit (Android)](https://manichord.com/projects/mgit.html).
+
+Linking notes is supported in the [same fashion that Dendron uses](https://wiki.dendron.so/notes/9MZBqhrijEM4QpZRa5t08/) (ex. `[[my.other.note]]`), which is a nice plus.
+
+Paid options can assist note-takers with automatic syncing, publishing of notes, and other perks.
 
 ## GitJournal
 
 ![Editor and markdown-preview screenshots, on Android, for GitJournal](https://org-dendron-public-assets.s3.amazonaws.com/images/blog-gitjournal-screenshots.png)
 
+> _"Mobile first Markdown Notes integrated with Git."_
+
 - **Cost:** Free (or a [paid Pro option](https://gitjournal.io/pricing/))
 - **OS:** Android, iOS
 - **Open Source:** Yes ([AGPL-3.0](https://github.com/GitJournal/GitJournal/blob/master/LICENSE))
-- **Git integration:** Yes
+- **Built-in git integration:** Yes
 
-[GitJournal](https://gitjournal.io/) has floated to the top of my list: works on mobile device of choice, free to jump into, open to contributions, and can sync with my repos. The project is built with [Flutter](https://flutter.dev/), making life easier when it comes to multiple-platforms as target exports.
+_[GitJournal](https://gitjournal.io/)_ has gained quite a bit of interest: it works on your mobile device of choice, free to jump into, open to contributions, and can sync notes to a repo (or multiple repos with the Pro version). The project is built with [Flutter](https://flutter.dev/), making life easier when it comes to multiple-platforms as target exports.
 
-I'm really interested in seeing where GitJournal goes. I couldn't find other mobile apps that covered both notes in Markdown and git-integration, without needing to juggle multiple apps. With my [Dendron vaults](https://wiki.dendron.so/notes/6682fca0-65ed-402c-8634-94cd51463cc4.html), I mostly take [scratch notes](https://wiki.dendron.so/notes/5c213aa6-e4ba-49e8-85c5-1bdcb33ce202.html#scratch-note) or edit already-existing notes when using my phone. For more extensive usage, I hop on the laptop.[^1]
+Linking notes is supported in the same fashion that Dendron uses, like Obsidian.
```

Comment:
GitJournal also supports wikilinks and can therefore be used with Dendron and Obsidian.